frontend/src/metabase/admin/people/components/GroupDetail.jsx

```diff
@@ -173,38 +78,98 @@ export default class GroupDetail extends Component {
           )}
         </span>
       </React.Fragment>
-    );
-
-    return (
-      <AdminPaneLayout
-        title={title}
-        buttonText={t`Add members`}
-        buttonAction={
-          canEditMembership(group) ? this.onAddUsersClicked.bind(this) : null
-        }
-        buttonDisabled={addUserVisible}
-      >
-        <GroupDescription group={group} />
-        <GroupMembersTable
-          currentUser={currentUser}
-          group={group}
-          members={members}
-          users={filteredUsers}
-          showAddUser={addUserVisible}
-          text={text || ""}
-          selectedUsers={selectedUsers}
-          onAddUserCancel={this.onAddUserCanceled.bind(this)}
-          onAddUserDone={this.onAddUserDone.bind(this)}
-          onAddUserTextChange={this.onAddUserTextChange.bind(this)}
-          onUserSuggestionAccepted={this.onUserSuggestionAccepted.bind(this)}
-          onRemoveUserFromSelection={this.onRemoveUserFromSelection.bind(this)}
-          onRemoveUserClicked={this.onRemoveUserClicked.bind(this)}
-        />
-        <Alert
-          message={alertMessage}
-          onClose={() => this.setState({ alertMessage: null })}
-        />
-      </AdminPaneLayout>
-    );
-  }
-}
+    ),
+    [group, members],
+  );
+
+  const onRemoveUserClicked = useCallback(
```

Comment:
nit: we use `handle` for callbacks and `on` for props. Here it would be `handleRemoveUserClick`